api/src/damnit_api/db.py

```diff
@@ -100,27 +100,8 @@ async def async_variables(proposal):
         result = await session.execute(selection_variables)
 
     variable_rows = result.mappings().all()
-    variables_list = [dict(r) for r in variable_rows]
 
-    try:
-        variable_tags = await async_table(proposal, name="variable_tags")
-        selection_tags = select(variable_tags.c.variable_name, variable_tags.c.tag_id)
-        async with get_session(proposal) as session:
-            tags_result = await session.execute(selection_tags)
-
-        tags_list = tags_result.mappings().all()
-        tags_map: dict[str, list[int]] = {}
-        for row in tags_list:
-            name = row["variable_name"]
-            tag_id = row["tag_id"]
-            tags_map.setdefault(name, []).append(tag_id)
-    except NoSuchTableError:
-        tags_map = {}
-
-    for var in variables_list:
-        var["tag_ids"] = tags_map.get(var["name"], [])
-
-    return create_map(variables_list, key="name")
+    return [dict(r) for r in variable_rows]
```

Comment:
I believe this is an artifact from the last refactoring: the function is expected to return a map.

(This reminds me to spray annotations everywhere XD)